.changeset/gold-dingos-jump.md

```diff
@@ -0,0 +1,7 @@
+---
+'@chromatic-com/playwright': patch
+'@chromatic-com/cypress': patch
+'@chromatic-com/shared-e2e': patch
+---
+
+Hashes Reserved Storybook files so they are not overwritten
```

Comment:
Super minor but it may be more clear to say that we're renaming the files rather than hashing them.